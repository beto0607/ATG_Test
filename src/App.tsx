import React from 'react';
import Layout from './layout/Layout';

const App: React.FC = () => (
  <Layout>
    <div>Wacala</div>
  </Layout>
);


export default App;
