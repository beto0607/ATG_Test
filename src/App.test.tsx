import React from 'react';
import { shallow } from 'enzyme';
import App from './App';
import toJson from 'enzyme-to-json'

describe('App', () => {
  it('renders without crashing without props', () => {
    const wrapper = shallow(<App />);
    expect(toJson(wrapper)).toMatchSnapshot()
  });
  it('renders without crashing with props', () => {
    const props = { any: 'any' };
    const wrapper = shallow(<App {...props} />);
    expect(toJson(wrapper)).toMatchSnapshot()
  });
});