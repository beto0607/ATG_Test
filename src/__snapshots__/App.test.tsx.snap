// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App renders without crashing with props 1`] = `
<Provider
  store={
    Object {
      "dispatch": [Function],
      "getState": [Function],
      "replaceReducer": [Function],
      "subscribe": [Function],
      Symbol(observable): [Function],
    }
  }
>
  <Layout>
    <Connect(ConnectedForm) />
    <Connect(ConnectedInfoSection) />
  </Layout>
</Provider>
`;

exports[`App renders without crashing without props 1`] = `
<Provider
  store={
    Object {
      "dispatch": [Function],
      "getState": [Function],
      "replaceReducer": [Function],
      "subscribe": [Function],
      Symbol(observable): [Function],
    }
  }
>
  <Layout>
    <Connect(ConnectedForm) />
    <Connect(ConnectedInfoSection) />
  </Layout>
</Provider>
`;
